modules/mssql/outputs.tf

```diff
@@ -61,6 +61,17 @@ output "generated_user_password" {
   sensitive   = true
 }
 
+output "additional_users" {
+  description = "Map of additional users and passwords"
+  value = [for r in google_sql_user.additional_users :
```

Comment:
This output is a list, not a map (suggest making it a map).

Reply:
I updated the output description on every module on the next commit. I did it this way to make it consistent with the input variable `additional_users`, so the input and the outputs have the same format.